parachain/src/cli.rs

```diff
@@ -44,43 +44,11 @@ pub enum Subcommand {
     Benchmark(frame_benchmarking_cli::BenchmarkCmd),
 }
 
-/// Command for exporting the genesis state of the parachain
-#[derive(Debug, Parser)]
-pub struct ExportGenesisStateCommand {
-    /// Output file name or stdout if unspecified.
-    #[clap(parse(from_os_str))]
-    pub output: Option<PathBuf>,
-
-    /// Write output in binary. Default is to write in hex.
-    #[clap(short, long)]
-    pub raw: bool,
-
-    /// The name of the chain for that the genesis state should be exported.
-    #[clap(long)]
-    pub chain: Option<String>,
-}
-
-/// Command for exporting the genesis wasm file.
-#[derive(Debug, Parser)]
-pub struct ExportGenesisWasmCommand {
-    /// Output file name or stdout if unspecified.
-    #[clap(parse(from_os_str))]
-    pub output: Option<PathBuf>,
-
-    /// Write output in binary. Default is to write in hex.
-    #[clap(short, long)]
-    pub raw: bool,
-
-    /// The name of the chain for that the genesis wasm file should be exported.
-    #[clap(long)]
-    pub chain: Option<String>,
-}
-
 /// Command for exporting the metadata.
 #[derive(Debug, Parser)]
 pub struct ExportMetadataCommand {
     /// Output file name or stdout if unspecified.
-    #[clap(parse(from_os_str))]
+    #[clap(action)]
```

Comment:
Why?

Reply:
Change in `clap-rs` [`v3.2.0`](https://github.com/clap-rs/clap/compare/v3.1.18...v3.2.0) [here](https://github.com/clap-rs/clap/pull/3794). See also the change in `cumulus` [here](https://github.com/paritytech/cumulus/commit/2e52ce29e91bcf34b3003810846c576bd4396575).

Reply:
That didn't really tell me what exactly it did, so I did some googling and found this explanation for what `action` does: https://github.com/clap-rs/clap/discussions/4041